File: src/KubeOps.Templates/Templates/Operator.CSharp/Startup.cs
using KubeOps.Operator;
using Microsoft.AspNetCore.Builder;
using Microsoft.Extensions.DependencyInjection;

namespace GeneratedOperatorProject
{
    public class Startup
    {
        public void ConfigureServices(IServiceCollection services)
        {
            services.AddKubernetesOperator();
        }

        public void Configure(IApplicationBuilder app)
        {
            app.UseKubernetesOperator();
        }
    }
}

File: _old/src/KubeOps.Templates/Templates/Operator.FSharp/Startup.fs
namespace GeneratedOperatorProject

open Microsoft.AspNetCore.Builder
open Microsoft.Extensions.DependencyInjection
open KubeOps.Operator

type Startup() =

    member _.ConfigureServices(services: IServiceCollection) =
        services.AddKubernetesOperator() |> ignore
        ()

    member _.Configure(app: IApplicationBuilder) =
        app.UseKubernetesOperator()
        ()
